src/tensor_product_operators.jl

```diff
@@ -171,3 +171,8 @@ function tensor_product_operator_2D(D_x, D_y = D_x)
                                                        acc_order, source)
     return TensorProductOperator(D_multi, N_x, N_y)
 end
+
+function Base.kron(D_x::AbstractNonperiodicDerivativeOperator,
```

Comment:
Does this follow the same memory layout logic?

Reply:
You mean compared to, e.g., `kron(::AbstractMatrix, ::AbstractMatrix)`? I'm not sure. Can you more concrete what should be the same?
If it's problematic to add a method to `Base.kron`, I can also delete it again. It would only be for convenience.

Reply:
If `sparse(kron(D1, D2)) == kron(sparse(D1), sparse(D2))`, I am fine with this convenience feature. Otherwise, I would like to avoid confusion (and maybe add a warning to the docstring of the te sor product operator).

Reply:
Sounds reasonable. I removed it again in [d4b387a](https://github.com/ranocha/SummationByPartsOperators.jl/pull/330/commits/d4b387a710673db1744457efafd7008a78bc6041).